hack/e2e/run-rp-and-e2e.sh

```diff
@@ -68,14 +80,19 @@ register_sub() {
       "https://localhost:8443/subscriptions/$AZURE_SUBSCRIPTION_ID?api-version=2.0"
 }
 
-clean_e2e_db(){
+clean_e2e_db() {
     echo "########## 🧹 Deleting DB $DATABASE_NAME ##########"
     az cosmosdb sql database delete --name $DATABASE_NAME \
         --yes \
         --account-name $DATABASE_ACCOUNT_NAME \
         --resource-group $RESOURCEGROUP >/dev/null
 }
 
+delete_e2e_cluster() {
+    echo "########## 🧹 Deleting Cluster $CLUSTER ##########"
+    CLUSTER="$CLUSTER" go run ./hack/cluster delete
```

Comment:
No need to re-assign `CLUSTER` to itself, the go script will pick it up if it's already on the environment.